ATGMediaBrowser/MediaBrowser/MediaBrowserViewController.swift

```diff
@@ -192,14 +210,20 @@ public class MediaBrowserViewController: UIViewController {
             contentViews.forEach({ $0.updateTransform() })
         }
     }
+    /// Variable to set title style in media browser.
+    public var titleStyle: TitleStyle? = TitleStyle(font: UIFont.preferredFont(forTextStyle: .subheadline), textColor: .white) {
```

Comment:
Make it non-optional please. No valid case for making it optional.

Reply:
`public var titleStyle = TitleStyle() {`is good enough since we have defined default variables inside the initializer.